### DISCUSSION TITLE
how to replace route 53 with godaddy in terraform script for hosted zones in serverpod project

### DISCUSSION THREAD

**Discussion by Shubhamdsingh** [serverpod/serverpod#411] [Q&A] [answered]
Can anyone help me to replace route 53 with godaddy in terraform script in serverpod project

**Comment by yahu1031** [comment]
@Shubhamdsingh Please raise discussions if they are questions.

BTW [watch this video](https://youtu.be/zx2eh3t_98Q?t=473)

@vlidholt discussuions please

**Comment by vlidholt** [comment, marked as answer]
Typically, even if you use Godaddy as you domain registrar, you will want to use Route 53. You just point your domain name servers to Route 53 and add the records in Route 53. If you already have a set of records configured in Godaddy, the easiest transistion is likely to recreate them in Routee 53. Not sure if it is possible to add the correct records needed for Serverpod's terraform scripts in Godaddy, it might be somehow.
